parser-typechecker/src/Unison/Server/Backend.hs

```diff
@@ -195,12 +204,23 @@ findShallowInBranch codebase b = do
       b0 = Branch.head b
   termEntries <- for (R.toList . Star3.d1 $ Branch._terms b0) $ \(r, ns) -> do
     ot <- lift $ loadReferentType codebase r
-    pure $ ShallowTermEntry r (hqTerm b0 ns r) ot
+    let isDoc = case ot of
+          Just t  -> Typechecker.isSubtype t $ Type.ref mempty Decls.docRef
```

Comment:
A term is a doc if its type conforms to `Doc`

Reply:
These comments might be better as code comments, wdyt?